ddtrace/contrib/django/patch.py

```diff
@@ -400,6 +366,37 @@ def traced_get_response(django, pin, func, instance, args, kwargs):
             # Set HTTP Request tags
             response = func(*args, **kwargs)
 
+            try:
```

Comment:
why does moving this improve things?

also, does that mean we can end up with scenarios where we do not resolve the requested url + resource name?

e.g. what happens if `func()` raises an exception (can it?)